build/mips-simd-check.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
is this scrip just for check mmi/msa which is the architecture of loongson.

Reply:
yes